auditwheel/policy/policy.json

```diff
@@ -40,6 +40,32 @@
          "GCC": ["3.0", "3.3", "3.3.1", "3.4", "3.4.2", "3.4.4", "4.0.0",
                  "4.2.0", "4.3.0"]
      },
+     "lib_whitelist": [
+         "libgcc_s.so.1",
+         "libstdc++.so.6",
+         "libm.so.6", "libdl.so.2", "librt.so.1",
+         "libc.so.6", "libnsl.so.1", "libutil.so.1", "libpthread.so.0",
+         "libX11.so.6", "libXext.so.6", "libXrender.so.1", "libICE.so.6",
+         "libSM.so.6", "libGL.so.1", "libgobject-2.0.so.0",
+         "libgthread-2.0.so.0", "libglib-2.0.so.0", "libresolv.so.2"
+     ]},
+    {"name": "manylinux2014",
+     "priority": 80,
+     "symbol_versions": {
+         "GLIBC": ["2.0", "2.1", "2.1.1", "2.1.2", "2.1.3", "2.2", "2.2.1",
+                   "2.2.2", "2.2.3", "2.2.4", "2.2.5", "2.2.6", "2.3", "2.3.2",
+                   "2.3.3", "2.3.4", "2.4", "2.5", "2.6", "2.7", "2.8", "2.9",
+                   "2.10", "2.11", "2.12", "2.13", "2.14", "2.15", "2.16",
+                   "2.17"],
+         "CXXABI": ["1.3", "1.3.1", "1.3.2", "1.3.3", "1.3.4", "1.3.5",
+                    "1.3.6", "1.3.7", "TM_1"],
+         "GLIBCXX": ["3.4", "3.4.1", "3.4.2", "3.4.3", "3.4.4", "3.4.5",
+                     "3.4.6", "3.4.7", "3.4.8", "3.4.9", "3.4.10", "3.4.11",
+                     "3.4.12", "3.4.13", "3.4.14", "3.4.15", "3.4.16",
+                     "3.4.17", "3.4.18", "3.4.19"],
+         "GCC": ["3.0", "3.3", "3.3.1", "3.4", "3.4.2", "3.4.4", "4.0.0",
+                 "4.2.0", "4.3.0", "4.7.0", "4.8.0"]
```

Comment:
PEP 599 goes up to `GCC_4.8.5`. Should we add that too?

Reply:
I think that PEP shall be updated. I'll open a PR.
here are the results for i686/aarch64 architectures (note the presence of `TM_1` here too):
i686:
```
{
"GLIBC": ["2.0", "2.1", "2.1.1", "2.1.2", "2.1.3", "2.2", "2.2.1", "2.2.2", "2.2.3", "2.2.4", "2.2.6", "2.3", "2.3.2", "2.3.3", "2.3.4", "2.4", "2.5", "2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "2.12", "2.13", "2.14", "2.15", "2.16", "2.17"],
"CXXABI": ["1.3", "1.3.1", "1.3.2", "1.3.3", "1.3.4", "1.3.5", "1.3.6", "1.3.7", "TM_1"],
"GLIBCXX": ["3.4", "3.4.1", "3.4.2", "3.4.3", "3.4.4", "3.4.5", "3.4.6", "3.4.7", "3.4.8", "3.4.9", "3.4.10", "3.4.11", "3.4.12", "3.4.13", "3.4.14", "3.4.15", "3.4.16", "3.4.17", "3.4.18", "3.4.19"],
"GCC": ["3.0", "3.3", "3.3.1", "3.4", "3.4.2", "4.0.0", "4.2.0", "4.3.0", "4.4.0", "4.5.0", "4.7.0", "4.8.0"]
}
```
aarch64:
```
{
"GLIBC": ["2.0", "2.17", "2.18"],
"CXXABI": ["1.3", "1.3.1", "1.3.2", "1.3.3", "1.3.4", "1.3.5", "1.3.6", "1.3.7", "TM_1"],
"GLIBCXX": ["3.4", "3.4.1", "3.4.2", "3.4.3", "3.4.4", "3.4.5", "3.4.6", "3.4.7", "3.4.8", "3.4.9", "3.4.10", "3.4.11", "3.4.12", "3.4.13", "3.4.14", "3.4.15", "3.4.16", "3.4.17", "3.4.18", "3.4.19"],
"GCC": ["3.0", "3.3", "3.3.1", "3.4", "3.4.2", "3.4.4", "4.0.0", "4.2.0", "4.3.0", "4.5.0", "4.7.0"]
}
```

Reply:
Good catch, that's a typo. GCC 4.8.5 uses the `GCC_4.8.0` symbol version. I've made the change in https://github.com/python/peps/pull/1181.